src/Dfc.CourseDirectory.WebV2/Features/Courses/DeleteCourseRun.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Dfc.CourseDirectory.Core;
+using Dfc.CourseDirectory.Core.DataStore.CosmosDb;
+using Dfc.CourseDirectory.Core.DataStore.CosmosDb.Models;
+using Dfc.CourseDirectory.Core.DataStore.CosmosDb.Queries;
+using Dfc.CourseDirectory.Core.Models;
+using Dfc.CourseDirectory.Core.Validation;
+using Dfc.CourseDirectory.WebV2.Behaviors;
+using FluentValidation.Results;
+using MediatR;
+using OneOf;
+using DeleteCourseRunQuery = Dfc.CourseDirectory.Core.DataStore.CosmosDb.Queries.DeleteCourseRun;
```

Comment:
Having to rename in a `using` is in my opinion a code smell, in as much as name clashes have a tendency to result in bugs and misunderstandings.

Within a single codebase I prefer slightly less concise naming to avoid having these clashes.

Reply:
If I change the naming of any of these types then they will be inconsistent with everything else. I'm not wild about the clash either but I'm more concerned about being inconsistent. Perhaps one to look at holistically across the project?